cluster/saltbase/salt/flannel-server/network.json

```diff
@@ -0,0 +1,8 @@
+{
+    "Network": "192.168.0.0/16",
```

Comment:
I'm going with 172 for now because that's a lot easier to debug. I think there's something fishy with node port if everything (services flannel and nodes) uses the 10 dot space that I need to get to the bottom of.